ui/admin/config/environment.js

```diff
@@ -125,7 +125,10 @@ module.exports = function (environment) {
     };
 
     // Enable features in development
+    ENV.featureFlags['search'] = true;
```

Comment:
I don't think we need `search`. When I search the codebase I don't see it being used at all. Should we just remove it entirely?

Reply:
You are right @cameronperera , I assumed we still using it. I will remove it. Nice catch!! :raised_hands: